File: SofiForce.Models/Models/ListModels/VendorListModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class VendorListModel
    {
        public int VendorId { get; set; }
        public string VendorCode { get; set; }
        public int? VendorGroupId { get; set; }
        public string VendorName { get; set; }
        public bool? IsLocal { get; set; }
        public bool? IsActive { get; set; }
        public int? DisplayOrder { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string VendorGroupName { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/VendorListModel.ts
export class VendorListModel {
    vendorId: number;
    vendorCode: string;
    vendorGroupId: number | null;
    vendorName: string;
    isLocal: boolean | null;
    isActive: boolean | null;
    displayOrder: number | null;
    color: string;
    icon: string;
    canEdit: boolean | null;
    canDelete: boolean | null;
    vendorGroupName: string;
}